lib/spack/spack/installer.py

```diff
@@ -2034,6 +2034,9 @@ def _real_install(self):
 
                         # Catch any errors to report to logging
                         self.timer.start(phase_fn.name)
+                        import time
+
+                        time.sleep(1)
```

Comment:
What's the error here again? I know we went through this a little while ago, but I'd like to record that conversation here. Also can we sleep for less time?

Reply:
The error was when running `test_cdash_upload_build_error`. This test is skipped in this PR but this change was more here for the discussion.

 The installer would output the line [Executing phase install](https://github.com/spack/spack/blob/42a452d54c8a25f9f415fef8cf9e3a5c64b7a46a/lib/spack/spack/installer.py#L2031) after the actual install output. This makes cdash's build output reader fail because the build output is out of order.